.trivyignore

```diff
@@ -1 +1,2 @@
 CVE-2017-20165
+CVE-2023-2251
```

Comment:
can we put comments after these so we know the context of what/why we're ignoring and when to reassess ?